src/eckit/log/SysLog.h

```diff
@@ -99,6 +99,11 @@ class SysLog {
         return s;
     }
 
+    /// Optional fields for structured data (RFC 5424 section 6.3)
+    void software(const std::string& software) { software_ = software; }
```

Comment:
Where are these being set?

Reply:
These are the keywords that belongs to origin part of the syslog structural data described in RFC5424 -> https://datatracker.ietf.org/doc/html/rfc5424#section-7.2